src/commands/env/var/unset.ts

```diff
@@ -92,26 +91,17 @@ export default class ConfigUnset extends Command {
       };
     }, {});
 
+    const message = `Unsetting ${Object.keys(configPairs)
+      .map((key) => herokuColor.configVar(key))
+      .join(', ')} and restarting ${herokuColor.app(targetCompute)}`;
+    cli.action.start(message);
+
     await this.client.patch(`/apps/${appName}/config-vars`, {
       data: configPairs,
     });
 
-    if (!flags.json) {
-      cli.action.start(
-        `Unsetting ${Object.keys(configPairs)
-          .map((key) => herokuColor.configVar(key))
-          .join(', ')} and restarting ${herokuColor.app(targetCompute)}`
-      );
+    cli.action.stop();
 
-      cli.action.stop();
-    }
-
-    if (flags.json) {
-      cli.styledJSON({
-        status: 0,
-        result: null,
-        warnings: [],
-      });
-    }
+    return [];
```

Comment:
I won't hold up this PR for this but you should return _something_.
For these types of commands we often return the thing/object that was unset.